ssl/ssl_asn1.c

```diff
@@ -194,7 +194,7 @@ int i2d_SSL_SESSION(SSL_SESSION *in, unsigned char **pp)
     }
     if (in->ext.tick_lifetime_hint > 0)
         as.tlsext_tick_lifetime_hint = in->ext.tick_lifetime_hint;
-    as.tlsext_tick_age_add = in->ext.tick_age_add;
+    as.tlsext_tick_age_add = (long)(int32_t)in->ext.tick_age_add;
```

Comment:
Because I wanted to make the double cast explicit rather than implicit, i.e. it is intentional to cast to an int32_t first and then store it in a long.